screen/model/commandrouter.js

```diff
@@ -30,8 +30,13 @@ module.exports = class CommandRouter {
 
     createHandler(msg){
         let entityId = msg.entity_id;
-        this.handlers[entityId] = this.createHandlerFromType(entityId, msg.type);
-        this.sendMessageToHandler(entityId, msg);
+
+        if (!(entityId in this.handlers)){
```

Comment:
I think it would be more useful to delete the old and create a new one. Say Core wants to create an image at 50,50 but it already exists at 0,0. Then it's better if the result of the message is an image at 50,50 rather than 0,0. Or is this never meant to happen?